src/sass/modules/banner/_banner.scss

```diff
@@ -1,3 +1,51 @@
+$content-width: 1200px !default;
+$global-banner-height: 3.5rem;
+$global-banner-height_phone: 4rem;
+
+/* Shared Banner Styles
+  =========================================================================== */
+@mixin shared-banner {
```

Comment:
I'm not totally sure from the naming what a "shared" banner is opposed to global.

Reply:
Valid... Created a mixin to share styles across multiple banners. 

How about `default-banner`?

Reply:
Would that be different than `.banner`? Is there a `.banner` class? We just don't use "default" in other shared elements

Reply:
yeah. I guess that really is what it should be. Not sure why I decided to go the mixin approach?